salesforce/10.11/modules/ROOT/pages/salesforce-connector-upgrade-migrate.adoc

```diff
@@ -2,19 +2,23 @@
 :page-aliases: connectors::salesforce/salesforce-connector-100-upgrade-migrate.adoc
 
 
-Upgrade Anypoint Connector for Salesforce (Salesforce Connector) from version 9.x or 10.x to version 10.10.
+Upgrade Anypoint Connector for Salesforce (Salesforce Connector) from version 9.x or 10.x to version 10.11.
 
 == Supported Upgrade Paths
 
 [%header,cols="50a,50a"]
 |===
 |From Version | To Version
-|9.x |10.10
-|10.x |10.10
+|9.x |10.11
+|10.x |10.11
 |===
 
 == Changes
 
+=== 10.11
+
+* The *Query Result Stream* operation is deprecated. There are two new operations, *Query Result List* and *Query Result Stream By Id*, that should be used instead.
```

Comment:
If you have time, you might want to consider putting each new operation in a bullet and then having a short description of each one (you can probably lift the first line from the Reference).  Something like this:

Use one of the following new operations instead:

* *Query Result List* to xxx 
*  *Query Result Stream by Id* to xxxx

This is not a must do.